_episodes/06-loops-and-functions.md

```diff
@@ -12,7 +12,10 @@ objectives:
     - "Build reusable code in Python."
     - "Write functions using conditional statements (if, then, else)."
 keypoints:
-    - "FIXME"
+    - "Loops help automate repetitive tasks over sets of items."
+    - "Loops can be combined with functions we have learned so far to make data analysis and processing more efficient."
+    - "Conditional statements allow us to sort data and perform different operations on different subsets of the data."
```

Comment:
Conditional statements enable execution of different operations on different data.

Reply:
```suggestion
    - "Conditional statements enable execution of different operations on different data."
```